src/Numerical_aux/GaussianNucleationFunction.h

```diff
@@ -1,20 +1,24 @@
 #ifndef SEISSOL_GAUSSIANNUCELATIONFUNCTION_H
 #define SEISSOL_GAUSSIANNUCELATIONFUNCTION_H
 
+#include "Numerical_aux/Functions.h"
```

Comment:
IMHO there is no need to include this here. Should rather be included in the Dynamic Rupture classes.

Reply:
discussed